microk8s-resources/actions/common/utils.sh

```diff
@@ -485,6 +486,24 @@ render_csr_conf() {
     fi
 }
 
+compare_csr_conf() {
+    # Compares the file csr.conf.rendered with csr.conf
+    # Return
+    #  0 if no change
+    #  1 otherwise.
```

Comment:
This function will return "0 if no change" so when used in a bash if statement it will evaluate to true if there is no change. So if we have something like 
```
if compare_csr_conf ; then
  generate_cert
fi
```
we will generate a certificate if there is no change in the csr file. This does not sound right.

Reply:
Sorry I got confused, fixed.